README.md

```diff
@@ -26,31 +27,31 @@ If you are running Kodi Krypton (v17) the only way to install Retrospect is to i
+Starting from Kodi Leia (v18) the playback of DRM protected streams is supported using the `InputStream Adaptive` add-on. Kodi comes with this pre-installed, but by default it is disabled. Make sure that you **enable** it first. In order to play Widevine DRM files you will need to have the Google Widevine libraries installed. Android based devices have this as a native component, for Windows and Linux you will need to install them:
 
 The _Easy way_:
 
 1. Open the Retrospect add-on settings.
-1. Enable support for the InputStream Adaptive add-on within Retrospect by enabling `Use Kodi InputStream Adaptive add-on when possible`.
+1. Enable support for the InputStream Adaptive add-on by enabling `Use Kodi InputStream Adaptive add-on when possible`.
 1. Select **OK** to save the settings.
 1. Open the Retrospect add-on settings again.
-1. Go to `Install Widevine using InputStream Adaptive Helper add-on` en select this option.
-1. Complete the steps that the InputStream Adaptive Helper add-on give you.
+1. Go to `Install Widevine using InputStream Adaptive Helper add-on`.
+1. Complete the steps the InputStream Adaptive Helper add-on gives you.
 
 The _Manual way_:
 
-1. Determine the last version of the Widevine libraries: [https://dl.google.com/widevine-cdm/versions.txt](https://dl.google.com/widevine-cdm/versions.txt)
+1. Determine the latest version of the Widevine libraries: [https://dl.google.com/widevine-cdm/versions.txt](https://dl.google.com/widevine-cdm/versions.txt)
 1. Download the appropriate version for your OS/Kodi combination (replace the {version} with the most recent version):
     * 32-bit kodi on Windows: [https://dl.google.com/widevine-cdm/{version}-win-ia32.zip](https://dl.google.com/widevine-cdm/{version}-win-ia32.zip)
     * 64-bit kodi on Windows: [https://dl.google.com/widevine-cdm/{version}-win-x64.zip](https://dl.google.com/widevine-cdm/{version}-win-x64.zip)
     * 32-bit kodi on Linux: [https://dl.google.com/widevine-cdm/{version}-linux-ia32.zip](https://dl.google.com/widevine-cdm/{version}-linux-ia32.zip)
     * 32-bit kodi on Linux: [https://dl.google.com/widevine-cdm/{version}-linux-x64.zip](https://dl.google.com/widevine-cdm/{version}-linux-x64.zip)
-1. For Windows installation copy these files into your `<kodi-profile>\cdm` folder. Linux users need to install them manually (or they can use this [gist](https://gist.github.com/ruario/3c873d43eb20553d5014bd4d29fe37f1) ([Fork](https://gist.github.com/basrieter/44a463a97a60958c36435d54d50debb4)) to install it automatically).
+1. For Windows, copy these files into your `<kodi-profile>\cdm` folder. This is by default located in '%USERPROFILE%\Appdata\Roaming\Kodi\cdm. Linux users need to install them manually (or they can use this [gist](https://gist.github.com/ruario/3c873d43eb20553d5014bd4d29fe37f1) ([Fork](https://gist.github.com/basrieter/44a463a97a60958c36435d54d50debb4)) to install it automatically).
```

Comment:
This is incorrect. I run most Kodi installations portable, so it will not be in the `'%USERPROFILE%\Appdata\Roaming\Kodi\cdm'`

Reply:
True. Reverting.